app/src/organisms/GripperWizardFlows/UnmountGripper.tsx

```diff
@@ -70,15 +95,24 @@ export const UnmountGripper = (
       subHeader={t('please_retry_gripper_detach')}
       isSuccess={false}
     >
-      <Flex justifyContent={JUSTIFY_SPACE_BETWEEN} flex="1">
-        <Link
-          role="button"
-          css={TYPOGRAPHY.darkLinkH4SemiBold}
-          onClick={() => setShowGripperStillDetected(false)}
-        >
-          {t('shared:go_back')}
-        </Link>
-        <PrimaryButton onClick={handleContinue}>{t('try_again')}</PrimaryButton>
+      <Flex
+        width="100%"
+        justifyContent={JUSTIFY_SPACE_BETWEEN}
+        alignItems={ALIGN_FLEX_END}
```

Comment:
if ALIGN_FLEX_END is needed for desktop, probably need the following since the `Go back` and `Try again`'s vertical alignment is a little bit off.

```suggestion
        alignItems={isOnDevice ? ALIGN_CENTER ? ALIGN_FLEX_END}
```